README.md

```diff
@@ -1,10 +1,10 @@
-# Standard SQ-Scanner-based project analyzed on SonarCloud using Travis
+# Standard SQ-Scanner-based project analyzed on SonarQube Cloud using Travis
 
 [![Build status](https://travis-ci.org/SonarSource/sq-com_example_standard-sqscanner-travis.svg?branch=master)](https://travis-ci.org/SonarSource/sq-com_example_standard-sqscanner-travis) [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=com.sonarqube.examples.standard-sqscanner-travis-project&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=com.sonarqube.examples.standard-sqscanner-travis-project)
 
-#### This project is analysed on [SonarCloud](https://sonarcloud.io)!
+#### This project is analysed on [SonarQube Cloud](https://www.sonarsource.com/products/sonarcloud/)!

```

Comment:
I didn't consider updating product links; but makes sense to land on the proper marketing page